### PR TITLE
setup js/css build

### PR DESCRIPTION
**Pull request by herra**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    "node-sass": "^4.13.1",
+    "postcss-loader": "^3.0.0",
+    "sass-loader": "^8.0.2",
+    "script-loader": "^0.7.2",
+    "style-loader": "^1.1.3",
+    "webpack": "^4.42.0",
+    "webpack-dev-server": "^3.10.3"
+  },
+  "devDependencies": {
+    "@webpack-cli/serve": "^0.2.0",
```
Comment (groupsky) [inline package.json]: what is this used for?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    "@babel/preset-react": "^7.9.1",
+    "babel-loader": "^8.1.0",
+    "bootstrap": "^4.4.1",
+    "command-line-args": "^5.1.1",
+    "css-loader": "^3.4.2",
+    "mini-css-extract-plugin": "^0.9.0",
+    "node-sass": "^4.13.1",
+    "postcss-loader": "^3.0.0",
+    "sass-loader": "^8.0.2",
+    "script-loader": "^0.7.2",
```
Comment (groupsky) [inline package.json]: pre-stable version?

File: rt_dashboard/javascript/styles/main.scss [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+}
+
+table tr.failed td {
+  background-color: lightgoldenrodyellow;
+}
+
+span.loading {
+  font-size: .8em;
+  color: #555;
+}
```
Comment (groupsky) [inline rt_dashboard/javascript/styles/main.scss:123]: Where did all these styles come from?

File: package.json [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  "version": "1.0.0",
+  "description": "Redis Tasks Dashboard simple interface to manage and observe tasks",
+  "main": "webpack.config.js",
+  "directories": {
+    "test": "tests"
+  },
+  "scripts": {
+    "test": "test",
+    "build": "webpack",
+    "dev": "webpack-dev-server"
```
Comment (groupsky) [inline package.json]: maybe `start` would be better?

**Review by groupsky — CHANGES_REQUESTED** [review]
Based on #8, but doesn't seem necessary.

.editorconfig can be opened as a separate pr

I don't see a js linter

Add `.nvmrc` as well

File: rt_dashboard/javascript/styles/main.scss (hunk 3 above)
Comment (herra) [reply rt_dashboard/javascript/styles/main.scss:123]: From the current implementation. I am not sure whether to keep similarities with existing UI or to completely rewrite it.

**Review by groupsky — APPROVED** [review]

**Pull request merged**